test/software/amazon/ion/impl/bin/IonRawBinaryWriterTest.java

```diff
@@ -179,6 +179,15 @@ public NullDesc(final IonType type, final String literal)
         new NullDesc(STRUCT,    "null.struct")
     };
 
+    @Test
+    public void testFinishOnClose() throws IOException
```

Comment:
Should we also add a test to verify that calling `finish()` after `close()` does nothing?

Reply:
The first thing that happens in the test's call to `assertValue(...)` is `writer.finish()`, so I'd say the verification you're looking for (specifically:  no exception is thrown) is covered.

Reply:
Thanks for pointing that out. Looks good as-is.